3_javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>javascript3-week2</title>
+</head>
+
+<body>
+  <h2>javascript3-week2</h2>
+  <label for="amount"> Amount to Convert :</label><br>
+  <input type="text" id="amount" name="amount"><br>
+  <p id="rates"></p>
+  <div id="fromCurrencyContainer">
+    <p>From :</p>
+
+  </div>
+  <div id="toCurrencyContainer">
+    <p>To :</p>
+
+  </div>
+
+  <input type="button" value="Convert" id="mybtn">
```

Comment:
To make it easier to distinguish to elements, I would suggest you use a more descriptive ID, for example "convertBtn".